kernel/syscalls/file.md

```diff
@@ -1,6 +1,6 @@
 # File operations
 
-## DEPRECATED `syscalls_fileAdd` ⇾ `syscalls_fileOpen`
```

Comment:
About the changes in `file.md`, from what I see there are a few syscalls that are not available now and then new ones are described below. That's why we can remove it all, including a block of code (there is no such code now in our system)